judge/admin/contest.py

```diff
@@ -151,7 +153,7 @@ def get_queryset(self, request):
         if request.user.has_perm('judge.edit_all_contest'):
             return queryset
         else:
-            return queryset.filter(organizers__id=request.profile.id)
+            return queryset.filter(Q(authors__id=request.profile.id) | Q(curators__id=request.profile.id))
```

Comment:
```suggestion
            return queryset.filter(Q(authors=request.profile) | Q(curators=request.profile))
```
Seems I missed this on the first pass.